tanjun/context.py

```diff
@@ -596,15 +603,17 @@ def resolve_to_member(self) -> hikari.InteractionMember:
     def resolve_to_member(self, *, default: _T) -> typing.Union[hikari.InteractionMember, _T]:
         ...
 
-    def resolve_to_member(self, *, default: _T = ...) -> typing.Union[hikari.InteractionMember, _T]:
+    def resolve_to_member(
+        self, *, default: typing.Union[_T, _SENTINEL_TYPE] = _SENTINEL
```

Comment:
I had to replace `...` with a sentinel because you cannot type an ellipsis type (https://github.com/python/mypy/issues/11559) and mypy was complaining `...` wasn't part of `_T`: it only does its magic for functions with no body.

Reply:
I don't like this change, seems like it just introduces unnecessary slowdown to avoid what is just a standard pattern

Reply:
It's not *too* much slowdown (isinstance on type with minimal mro) but I see what you mean, I'll see if I can cludge mypy into liking it or something.

Reply:
its still worse than just using ellipsis though

Reply:
Came up with a better way of doing this, I think!